auth0/src/test/java/com/auth0/android/authentication/storage/CredentialsManagerTest.kt

```diff
@@ -170,30 +170,21 @@ public class CredentialsManagerTest {
         exception.expect(CredentialsManagerException::class.java)
         exception.expectMessage("Credentials must have a valid date of expiration and a valid access_token or id_token value.")
         val credentials: Credentials =
-            CredentialsMock(null, null, "type", "refreshToken", Date(), null)
-        manager.saveCredentials(credentials)
-    }
-
-    @Test
-    public fun shouldThrowOnSetIfCredentialsDoesNotHaveExpiresAt() {
-        exception.expect(CredentialsManagerException::class.java)
-        exception.expectMessage("Credentials must have a valid date of expiration and a valid access_token or id_token value.")
-        val credentials: Credentials =
-            CredentialsMock("idToken", "accessToken", "type", "refreshToken", null, "scope")
+            CredentialsMock("", "", "type", "refreshToken", Date(), null)
         manager.saveCredentials(credentials)
     }
 
     @Test
     public fun shouldNotThrowOnSetIfCredentialsHasAccessTokenAndExpiresAt() {
         val credentials: Credentials =
-            CredentialsMock(null, "accessToken", "type", "refreshToken", Date(), null)
+            CredentialsMock("", "accessToken", "type", "refreshToken", Date(), null)
         manager.saveCredentials(credentials)
     }
 
     @Test
     public fun shouldNotThrowOnSetIfCredentialsHasIdTokenAndExpiresAt() {
         val credentials: Credentials =
-            CredentialsMock("idToken", null, "type", "refreshToken", Date(), null)
+            CredentialsMock("idToken", "", "type", "refreshToken", Date(), null)
```

Comment:
this case will never happen again, as Credentials will always have these properties passed as empty here. I'd remove this test case.